src/main/java/se/diabol/jenkins/pipeline/sort/GenericComponentComparator.java

```diff
@@ -21,20 +21,21 @@
 import hudson.ExtensionPoint;
 import hudson.model.Describable;
 import hudson.model.Descriptor;
-import se.diabol.jenkins.pipeline.domain.Component;
+import se.diabol.jenkins.core.GenericComponent;
 import se.diabol.jenkins.pipeline.util.JenkinsUtil;
 
 import java.util.Comparator;
 
-public abstract class ComponentComparator implements Comparator<Component>, ExtensionPoint,
-        Describable<ComponentComparator> {
+public abstract class GenericComponentComparator implements Comparator<GenericComponent>,
+        ExtensionPoint,
```

Comment:
Changing name of an Extension point will potentially break external code.

Reply:
Yes, that crossed my mind. I prefer a name that better reflects the intent though. If we incorporate the major work on minimum requirement upgrades (PR 278) we need to clearly document changes so it might be acceptable to do this change at the same time.